cmd/libsnap-confine-private/tool.c

```diff
@@ -112,7 +115,13 @@ void sc_call_snap_update_ns_as_user(int snap_update_ns_fd, const char *snap_name
                      * with either SNAPD_DEBUG=0 or SNAPD_DEBUG=1, see that function
                      * for details. */
                     "SNAPD_DEBUG=x", xdg_runtime_dir_env, snap_real_home_env, NULL};
-    sc_call_snapd_tool_with_apparmor(snap_update_ns_fd, "snap-update-ns", apparmor, aa_profile, argv, envp);
+    /* keep the current identity */
+    sc_identity no_change_identity = {
```

Comment:
Similarly to how there is `sc_root_group_identity()` should there be a `sc_no_change_identity()` defined and implemented for consistency?

Reply:
I'll open a followup with this.